cfme/utils/dockerbot/pytestbase/setup.sh

```diff
@@ -183,13 +183,23 @@ if [ -n "$PROVIDER" ]; then
 fi
 export APPLIANCE=${APPLIANCE-"None"}
 log "appliance: $APPLIANCE"
+export IP_ADDRESS=$IP_ADDRESS
+log "ip_address: $IP_ADDRESS"
 export USE_SPROUT=${USE_SPROUT-"no"}
 log "use_sprout: $USE_SPROUT"
 
 # Now fill out the env yaml with ALL THE THINGS
-cat > $CFME_REPO_DIR/conf/env.local.yaml <<EOF
+rm -f $CFME_REPO_DIR/conf.env.local.yaml
+
+if [ "$USE_SPROUT" == "no" ];
+    HOSTNAME="$IP_ADDRESS"
+else
+    HOSTNAME="SPROUT_SHOULD_OVERRIDE_THIS"
+fi    
+
+cat >> $CFME_REPO_DIR/conf/env.local.yaml <<EOF
 appliances:
-  - hostname: $IP_ADDRESS
+  - hostname: $HOSTNAME
```

Comment:
adding on this we can actually grep for SPROUT_SHOULD_OVERRIDE_THIS in the sprout test and bail out correctly

Reply:
Good point, see latest commits